locals.tf

```diff
@@ -3,4 +3,20 @@ locals {
     scope      = "terraform-managed"
     repository = "jenkins-infra/azure-net"
   }
+
+  vpn = {
+    shorthostname = "vpn-test"
```

Comment:
Note: temporary subdomain, to be replaced by "vpn" when switching to this VM for vpn.jenkins.io